### DISCUSSION TITLE
Please package as Flatpak for Linux

### DISCUSSION THREAD

**Discussion by RasheedAZ** [goastian/midori-desktop#35] [Ideas]
Thanks for reviving this project. Can you please package it as a Flatpak app that works across most Linux distros and submit an official verified listing to Flathub?

The old version of the browser is there:
https://flathub.org/apps/org.midori_browser.Midori

**Comment by Ponchale** [comment]
We are working on the Flatpak we hope to launch it today or tomorrow friend
